randomizer/residual_envs/residual_fetch_pick_and_place_env.py

```diff
@@ -0,0 +1,90 @@
+from randomizer.controllers.pick_and_place_controller import get_pick_and_place_control
+# from .oscillating_pick_and_place_controller import get_pick_and_place_control
+
+import gym
+from gym import error, spaces, utils
+from gym.utils import seeding
+
+import numpy as np
+import time
+
+import pdb
+
+
+class MyFetchPickAndPlaceEnv(gym.Env):
```

Comment:
why is this the name?

Reply:
That's what they named in rpl_environments. I didn't change any of their stuff